pkg/mimirtool/commands/remote_read_test.go

```diff
@@ -150,3 +152,38 @@ func TestTimeSeriesIterator(t *testing.T) {
 	}
 
 }
+
+// See discussion in https://github.com/grafana/mimir/pull/5700 for detailed explanation
```

Comment:
Can you add a brief description instead of linking to this PR? It's easier to read the test if all the necessary context is in the same file.

It doesn't need to be very details. For example
```suggestion
// TestEarlyCommit writes samples of many series that don't fit into the same append commit. It makes sure that batching the samples into many commits doesn't cause the appends to advance the head block too far and make future appends invalid.
```